pkgs/applications/networking/seafile-client/default.nix

```
@@ -1,23 +1,42 @@
-{ mkDerivation, lib, fetchFromGitHub, pkg-config, cmake, qtbase, qttools
-, seafile-shared, jansson, libsearpc
-, withShibboleth ? true, qtwebengine }:
+{ mkDerivation
+, lib
+, fetchFromGitHub
+, pkg-config
+, cmake
+, qtbase
+, qttools
+, libuuid
+, seafile-shared
+, jansson
+, libsearpc
+, withShibboleth ? true
+, qtwebengine
+}:
 
 mkDerivation rec {
```

```suggestion
stdenv.mkDerivation rec {
```
See https://github.com/NixOS/nixpkgs/issues/180841